src/Modal/ModalPopup.tsx

```diff
@@ -77,35 +122,4 @@ function ModalPopup({
   );
 }
 
-ModalPopup.propTypes = {
-  /** Specifies the contents of the modal */
-  children: PropTypes.node.isRequired,
-  /** A callback function for when the modal is dismissed */
-  onClose: PropTypes.func.isRequired,
-  /** Is the modal dialog open or closed */
-  isOpen: PropTypes.bool.isRequired,
-  /** Prevent clicking on the backdrop or pressing Esc to close the modal */
-  isBlocking: PropTypes.bool,
-  /** Insert modal into a different location in the DOM */
-  withPortal: PropTypes.bool,
-  // This type: https://stackoverflow.com/questions/48007326/what-is-the-correct-proptype-for-a-ref-in-react
-  /** Specifies an element near which the modal should be displayed */
-  positionRef: PropTypes.oneOfType([
-    PropTypes.func,
-    PropTypes.shape({ current: PropTypes.shape({}) }),
-  ]),
```

Comment:
Wow, this `positionRef` type was just completely wrong, eh?  It does seem that it needs to be an HTMLElement, not a Ref object.

Reply:
Yes, it was a bit confusing. I would love to change the param name to `referenceElement` or something similar but we are already using it in a lot of places.
Thankfully, we are already passing in `HTMLElement` to ModalPopup in frontend-app-authoring.